### ISSUE TITLE
question about pre-processing with DenseNet 

### ISSUE THREAD

**Issue by Mahmood1968** [dell-ai-engineering/BigDL-ImageProcessing-Examples#9]
I had a look at this post https://medium.com/intuitionmachine/notes-on-the-implementation-densenet-in-tensorflow-beeda9dd1504
"At first, I implemented a solution that divides image by 255. All works fine, but a little bit worse, than results reported in the paper. Ok, next I’ve implemented per channel normalization… And networks began works even worse. It was not clear for me why. So I’ve decided mail to the authors. Thanks to Zhuang Liu that answered me and point to another source code that I missed somehow. After precise debugging, it becomes apparent that images should be normalized by mean/std of all images in the dataset(train or test), not by its own only." 
It seems there are two phases of image preprocessing based on this post 
1. Subtract the means and divided by std of images themselves 
2. subtract  the means and divided by the std of imagenet from all images
